lib/opam_build.mli

```diff
@@ -0,0 +1,11 @@
+(** Build and test all the opam packages in a given build context.
+    [~variant] is the variant of the "ocurrent/opam" image.
+    [~repo] is the ID of the repository-under-test on GitHub. *)
+val v :
+  docker:(module S.DOCKER_CONTEXT with type source = 'source) ->
+  pull_schedule:Current_cache.Schedule.t ->
```

Comment:
```suggestion
  schedule:Current_cache.Schedule.t ->
```
No need to expose this detail, I think.

Reply:
I made the change. But as I argued [above](https://github.com/ocurrent/ocaml-ci/pull/89#discussion_r371179171), I think this logic should go into `pipeline.ml` (it needs to be better defined to be in lib/)